lib/collections.js

```diff
@@ -26,12 +38,19 @@ var normalizedWords = [];
 
 // The words stored as keys of a dictionary
 var uniqueWordsAsKeys = {};
+for (var i = 0; i < normalizedWords.length; i++) {
+  uniqueWordsAsKeys[normalizedWords[i]] = 1;
+}
 
 // The keys pulled out into a list
 var uniqueWords = [];
 
 /* your code starts here */
-
+var i = 0;
+for (var k in uniqueWordsAsKeys) {
+  uniqueWords[i] = k;
+  i++;
+}
```

Comment:
This will work, but we already have a tool to append things to an array - `.push()`.